content/en/integrations/guide/azure-portal.md

```diff
@@ -0,0 +1,195 @@
+
+#### Install
+
+To install the Datadog Agent, select the appropriate VM, then click "Install Agent". The portal asks for confirmation to install the Agent with the default key. Select "OK" to begin installation. Azure shows the status as `Installing` until the Agent is installed and provisioned. After the Datadog Agent is installed, the status changes to `Installed`.
+
+#### Uninstall
+
+If the Datadog Agent was installed with the Azure VM extension, you can uninstall the Agent by selecting the appropriate VM, then click "Uninstall Agent".
+
+If the Agent was installed using a different method, you can not use the Datadog resource to deploy or remove the Agent, but information about the Agent is still reflected on this page.
+
+### App Service extension
+
+Select "App Service extension" in the left sidebar to see a list of app services in the subscription. On this page, you can install the Datadog extension on Azure App Services to enable APM tracing and custom metrics.
+
+For each app service, the following information is displayed:
+
+| Column            | Description                                                                                                                                                                  |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Resource name     | The app name                                                                                                                                                                 |
+| Resource status   | Whether the app service is stopped or running. The app service must be running to initiate install. If the app service is stopped, installing the Datadog Agent is disabled. |
+| App service plan  | The specific plan configured for the app service                                                                                                                             |
+| Extension version | The Datadog extension version number                                                                                                                                         |
+
+#### Install
+
+To install the Datadog extension, select the appropriate app, then click "Install Extension". The portal asks for confirmation to install the extension. Select "OK" to begin installation. This restarts your app and adds the following settings:
+
+- `DD_API_KEY:<DEFAULT_API_KEY>`
+- `DD_SITE:us3.datadoghq.com`
+- `DD_LOGS_INJECTION:true`
+
+Azure shows the status as `Installing` until the Agent is installed and provisioned. After the Datadog Agent is installed, the status changes to `Installed`.
+
+**Note**: Ensure you are adding the extension to apps with supported runtimes. The Datadog resource does not limit or filter the list of apps.
```

Comment:
let's link "supported runtimes" to https://docs.datadoghq.com/serverless/azure_app_services/#requirements